framework/db/conditions/AndCondition.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
```

Comment:
Standard Yii header missing.

Reply:
It will be added during release process automatically